src/main/java/frc/robot/Robot.java

```diff
@@ -119,6 +131,8 @@ public void teleopInit() {
   @Override
   public void teleopPeriodic() {
     CommandScheduler.getInstance().run();
+    DrivingDeltas calculatedDeltas = vision.targetDelta();
+      controls.getXButton().whenPressed(new LimelightTopTargetCommand(drive, limelight));
```

Comment:
Move this to the IO class (soon to be renamed Operator Inputs).

Reply:
Also make this a CommandGroup with ShooterCommand so when aiming is finished the power cells get launched.